src/iexamine.cpp

```diff
@@ -3094,6 +3104,12 @@ void (iexamine::*iexamine_function_from_string(std::string function_name))(playe
     if ("shrub_wildveggies" == function_name) {
         return &iexamine::shrub_wildveggies;
     }
+    if ("shrub_strawberry" == function_name) {
```

Comment:
Nothing calls these two, they were converted to use harvest_tree_shrub when that method was added, and only the declaration of the methods were left.